velox/dwio/parquet/tests/reader/ParquetTableScanTest.cpp

```diff
@@ -820,6 +821,125 @@ TEST_F(ParquetTableScanTest, timestampPrecisionMicrosecond) {
   assertEqualResults({expected}, result.second);
 }
 
+TEST_F(ParquetTableScanTest, readParquetColumnsByIndex) {
```

Comment:
@agrawalreetika Can you please add some tests for nested types like map and array?